flow/benchmarks/rllib/es_runner.py

```diff
@@ -33,11 +79,15 @@
 
     agent_cls = get_agent_class(alg_run)
     config = agent_cls._default_config.copy()
-    config["episodes_per_batch"] = N_ROLLOUTS
-    config["num_workers"] = N_ROLLOUTS
+    config["num_workers"] = min(num_cpus, num_rollouts)
+    config["episodes_per_batch"] = num_rollouts
     config["eval_prob"] = 0.05
     config["noise_stdev"] = grid_search([0.01, 0.02])
     config["stepsize"] = grid_search([0.01, 0.02])
+
+    config["noise_stdev"] = 0.02
```

Comment:
we should set the hp based on the best performing for each benchmark (see paper for details)